tsdb/chunkenc/histogram.go

```diff
@@ -780,22 +782,18 @@ func (a *HistogramAppender) AppendHistogram(prev *HistogramAppender, t int64, h
 			a.setCounterResetHeader(CounterReset)
 		case prev != nil:
 			// This is a new chunk, but continued from a previous one. We need to calculate the reset header unless already set.
-			_, _, _, _, _, counterReset := prev.appendable(h)
-			if counterReset {
-				a.setCounterResetHeader(CounterReset)
-			} else {
-				a.setCounterResetHeader(NotCounterReset)
-			}
+			_, _, _, _, _, counterResetHeader := prev.appendable(h)
```

Comment:
Even though we are using the return value to set the counter reset header, maybe `counterReset` is still a nicer and shorter name here.